spring-cloud-dataflow-configuration-metadata/src/main/java/org/springframework/cloud/dataflow/configuration/metadata/container/DefaultContainerImageMetadataResolver.java

```diff
@@ -48,12 +51,23 @@ public Map<String, String> getImageLabels(String imageName) {
 
 		ContainerRegistryRequest registryRequest = this.containerRegistryService.getRegistryRequest(imageName);
 
-		Map manifest = this.containerRegistryService.getImageManifest(registryRequest, Map.class);
-
-		if (manifest != null && !isNotNullMap(manifest.get("config"))) {
-			throw new ContainerRegistryException(
-					String.format("Image [%s] has incorrect or missing manifest config element: %s",
-							imageName, manifest.toString()));
+		Map<String, Object> manifest = this.containerRegistryService.getImageManifest(registryRequest, Map.class);
+
+		if (manifest != null && manifest.get("config") == null) {
+			// when both Docker and OCI images are stored in repository the response for OCI image when using Docker manifest type will not contain config.
+			// In the case where we don't receive a config and schemaVersion is less than 2 we try OCI manifest type.
+			String manifestMediaType = registryRequest.getRegistryConf().getManifestMediaType();
+			if (asInt(manifest.get("schemaVersion")) < 2
+					&& !manifestMediaType.equals(ContainerRegistryProperties.OCI_IMAGE_MANIFEST_MEDIA_TYPE)) {
```

Comment:
Is there another media type for schemaVersion == 1?

Reply:
There are only 2 media types. The default is Docker manifest. If it doesn't find the config then switching to accepting the OCI manifest should retrieve the info.

Reply:
Gotcha !   Thanks!